packages/common/src/ui/components/modals/BasicModal/BasicModal.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { FC } from 'react';
+import Dialog, { DialogProps as MuiDialogProps } from '@mui/material/Dialog';
+
+interface DialogProps extends MuiDialogProps {
+  height?: number;
+  width?: number;
+}
+
+export const BasicModal: FC<DialogProps> = ({
+  PaperProps,
+  width = 500,
+  height = 400,
+  ...dialogProps
+}) => {
+  return (
+    <Dialog
+      PaperProps={{
+        sx: {
+          borderRadius: '20px',
+          minHeight: `${height || '400'}px`,
+          minWidth: `${width || '500'}px`,
+        },
+        ...PaperProps,
+      }}
+      {...dialogProps}
+    >
+      {dialogProps.children}
+    </Dialog>
+  );
```

Comment:
Just the `Dialog` you had in `useDialog` from which I wanted to use, but I wanted to have the control one level higher than the hook so ended up taking this out to use in a different places - again not really much styling but nice to not have to redo it